apps/cowswap-frontend/src/modules/usdAmount/apis/getCoingeckoUsdPrice.ts

```diff
@@ -18,7 +18,7 @@ export const COINGECK_PLATFORMS: Record<SupportedChainId, string | null> = {
   [SupportedChainId.SEPOLIA]: null,
 }
 
-const BASE_URL = 'https://api.coingecko.com/api/v3/simple/token_price'
+const BASE_URL = 'https://cow-web-services-git-feat-usd-proxy-cowswap.vercel.app/api/serverless/coingeckoProxy'
```

Comment:
WARNING!!

This is the PR link.
Should switch to prod endpoint once that's merged, before this is one is merged

Reply:
Parent PR merged, replaced with the final endpoint.